### ISSUE TITLE
External component has thrown an exception.

### ISSUE THREAD

**Issue by math320** [RatScanner/RatScanner#23]
**Error**
External component has thrown an exception.
```
System.Runtime.InteropServices.SEHException (0x80004005): External component has thrown an exception.
   at OpenCvSharp.NativeMethods.text_OCRTesseract_create(String datapath, String language, String charWhitelist, Int32 oem, Int32 psmode, IntPtr& returnValue)
   at OpenCvSharp.Text.OCRTesseract.Create(String datapath, String language, String charWhitelist, Int32 oem, Int32 psmode)
   at RatScanner.Scan.ItemNameScan.Init()
   at RatScanner.RatScannerMain..ctor()
   at RatScanner.RatScannerMain.get_Instance()
   at RatScanner.View.MainMenu..ctor()
   at RatScanner.PageSwitcher..ctor()
```
<details>
<summary>Log</summary>

```
[13:47:10.4782719] > [Info]  ----- RatScanner 2.2.1 -----
[13:47:10.4794466] > [Info]  Starting RatScanner...
[13:47:10.4798370] > [Info]  Loading config...
[13:47:10.4943106] > [Info]  Loading price data...
[13:47:11.7315764] > [Info]  Setting default item...
[13:47:11.7364728] > [Info]  Initializing name scan...
[13:47:11.7368641] > [Info]  Loading OCR data...
[13:47:11.7595950] > [Error] External component has thrown an exception.
 -------------------- 
 System.Runtime.InteropServices.SEHException (0x80004005): External component has thrown an exception.
   at OpenCvSharp.NativeMethods.text_OCRTesseract_create(String datapath, String language, String charWhitelist, Int32 oem, Int32 psmode, IntPtr& returnValue)
   at OpenCvSharp.Text.OCRTesseract.Create(String datapath, String language, String charWhitelist, Int32 oem, Int32 psmode)
   at RatScanner.Scan.ItemNameScan.Init()
   at RatScanner.RatScannerMain..ctor()
   at RatScanner.RatScannerMain.get_Instance()
   at RatScanner.View.MainMenu..ctor()
   at RatScanner.PageSwitcher..ctor()
```
</details>

**Comment by math320** [comment]
Unable to run the program

**Comment by Blightbuster** [comment]
Hmm are you on a 32bit system?

**Comment by math320** [comment]
It is a 64-bit system
[cid:image003.png@01D69B09.927747F0]

**Comment by Blightbuster** [comment]
I created a issue in the repository of the used library:
shimat/opencvsharp#1058

**Comment by Blightbuster** [comment]
@math320 perhaps related to [this](https://github.com/Blightbuster/RatScanner/issues/28#issuecomment-719998661)?

Are you also using a non standard version of windows?

**Comment by Blightbuster** [comment]
Seems like this is a issue with the trained ocr model. Could you ( or any one else having this issue ) try to replace the current model (located at `RatScanner\Data\bender.traineddata`) with a new one I trained: [bender.zip](https://github.com/RatScanner/RatScanner/files/6760507/bender.zip) and report back if this solves the issue?

**Comment by Blightbuster** [comment]
Closing since this is ( presumably ) fixed with release v2.7.0. Please comment if this issue persists and I will re-open the issue.
